diff --git a/Makefile_h b/Makefile_h
index 91e0da6..bb432d3 100644
--- a/Makefile_h
+++ b/Makefile_h
@@ -1 +1 @@
-SYS_ARC = arm-linux-gnueabihf
+SYS_ARC = x86_64-linux-gnu
diff --git a/device_controller_api.cpp b/device_controller_api.cpp
deleted file mode 100644
index 57c2b22..0000000
--- a/device_controller_api.cpp
+++ /dev/null
@@ -1,6 +0,0 @@
-#include "device_controller_api.h"
-
-device_controller_api::device_controller_api()
-{
-
-}
diff --git a/device_controller_api.h b/device_controller_api.h
deleted file mode 100644
index 3a38b6d..0000000
--- a/device_controller_api.h
+++ /dev/null
@@ -1,11 +0,0 @@
-#ifndef DEVICE_CONTROLLER_API_H
-#define DEVICE_CONTROLLER_API_H
-
-
-class device_controller_api
-{
-public:
-    device_controller_api();
-};
-
-#endif // DEVICE_CONTROLLER_API_H
\ No newline at end of file
diff --git a/src/connect_init.h b/src/connect_init.h
index ced9830..b9e49af 100644
--- a/src/connect_init.h
+++ b/src/connect_init.h
@@ -21,8 +21,8 @@ private:
 	string sub_domain = "ialoy";
 	string domain = "arghyabiswas.com";
 	string api = "desktop_api";
-	string req_manager = "req_mngr.php";
-	string url = protocol+sub_domain+"."+domain+"/"+api+"/"+req_manager;
+	string set_up_req_manager = "set_up_req_mngr.php";
+	string url = protocol+sub_domain+"."+domain+"/"+api+"/"+set_up_req_manager;
 	string req_url;
 
 public:
